docs/book/driver-deployment/deploying_csi_with_zones.md

```diff
@@ -132,13 +132,12 @@ Install the vSphere CPI and the CSI driver using the zone and region entries.
    ```bash
    cd /etc/kubernetes
    kubectl create configmap cloud-config --from-file=vsphere.conf --namespace=kube-system
```

Comment:
`--namespace=kube-system` : Is this correct ?

Reply:
Shouldn't this be `vmware-system-csi`

Reply:
We will move `vmware-system-csi` in v2.3. This PR is just fixing some broken links in our documentation.